tests/datasource_tests.py

```diff
@@ -17,9 +17,10 @@
 """Unit tests for Superset"""
 import json
 from copy import deepcopy
+from typing import Dict
 
 from superset import db
-from superset.connectors.sqla.models import SqlaTable
+from superset.connectors.sqla.models import SqlaTable, TableColumn
```

Comment:
why did these get added if the rest of the file didn't change?

Reply:
Oh thanks for catching, I implemented the tests here first, but then decided the other test file was the correct place. These imports were left behind from that move.